test/e2e/autoscaling-max-tasks-per-minute/actor/package.json

```diff
@@ -6,7 +6,6 @@
         "apify": "^3.0.0-beta.75",
         "@apify/storage-local": "^2.1.0",
         "@crawlee/basic": "file:./packages/basic-crawler",
-        "@crawlee/browser-pool": "file:./packages/browser-pool",
```

Comment:
is this on purpose? why?

Reply:
I just copied from autoscaling test, but I'll remove it as its not needed

Reply:
Ohhh, I didn't check the test you reviewed this for. I think I just removed it because I noticed it wasn't needed, but that does go past the scope of the PR so I'l revert